packages/api/package.json

```diff
@@ -1,12 +1,15 @@
 {
   "name": "@atproto/api",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "main": "src/index.ts",
   "scripts": {
     "codegen": "lex gen-api ./src/client ../../lexicons/com/atproto/*/* ../../lexicons/app/bsky/*/*",
     "build": "node ./build.js",
     "postbuild": "tsc --build tsconfig.build.json",
     "update-main-to-dist": "node ./build.js --update-main-to-dist",
+    "update-main-to-src": "node ./build.js --update-main-to-src",
```

Comment:
we may need to rethink how these scripts work. we probably don't need to rebuild on `postpublish` & `prepublish` doesn't build types so those could get out of sync with the built code

don't think it needs to be polished yet, but maybe we open an issue so we don't lose track

Reply:
Good point, I'll open that issue